schematic/models/validate_attribute.py

```diff
@@ -857,26 +1039,57 @@ def url_validation(
                         errors.append(vr_errors)
                     if vr_warnings:
                         warnings.append(vr_warnings)
-                if valid_url == True:
-                    # If the URL works, check to see if it contains the proper arguments
-                    # as specified in the schema.
-                    for arg in url_args:
-                        if arg not in url:
-                            url_error = "arg_error"
-                            vr_errors, vr_warnings = GenerateError.generate_url_error(
-                                url,
-                                url_error=url_error,
-                                row_num=str(i + 2),
-                                attribute_name=manifest_col.name,
-                                argument=arg,
-                                invalid_entry=manifest_col[i],
-                                dmge=self.dmge,
-                                val_rule=val_rule,
-                            )
-                            if vr_errors:
-                                errors.append(vr_errors)
-                            if vr_warnings:
-                                warnings.append(vr_warnings)
+                else:
+                    # add scheme to the URL if not currently added.
+                    if not urlparse(url).scheme:
+                        url = "http://" + url
+                    try:
+                        # Check that the URL points to a working webpage
+                        # if not log the appropriate error.
+                        request = Request(url)
+                        response = urlopen(request)
+                        valid_url = True
+                        response_code = response.getcode()
+                    except:
```

Comment:
Do you want to catch every exception here?

Reply:
Just to note, i didnt write this section, Its appearing as a change bc I tab indented it bc I added an if. My guess is that if getting the response code doesnt work the idea is to raise an error.